src/util/util_statuscode.h

```diff
@@ -158,6 +158,12 @@ namespace dxvk {
         return NVAPI_INSUFFICIENT_BUFFER;
     }
 
+    inline NvAPI_Status NoActiveSliTopology(const std::string& logMessage, bool& alreadyLogged) {
+        if (log::tracing() || !std::exchange(alreadyLogged, true))
+            log::info(str::format("<-", logMessage, ": Error"));
```

Comment:
This should be `"NoActiveSLITopology"` instead of `"Error"`  to match the corresponding return code in the logs.

Reply:
Oops. That's what happens when you finish the commit at 2am.

Reply:
Fixed the log line and updated the commit.